simulation/src/curp/mod.rs

```diff
@@ -0,0 +1 @@
+pub mod curp_group;
```

Comment:
if there is only one module `curp_group` in the `curp`, why you use directory and `mod.rs`?

Reply:
I guess more files will be added later. But can you @bsbds explain your plan?

Reply:
As of my current plan, there's only one module needed. My design is: for each object, create a separate folder. For example, put all of curp's stuff in the `curp` folder, and for all of xline's (which will be added in the future), put them in the `xline` folder.

Is one file more preferable?

Reply:
IMO, what we truly need is a logical unit of division, as to whether it is a folder or a module doesn’t matter. At the aforementioned design, I prefer using a file.  We can change it to a folder if we really need it down the road.

Reply:
Makes sense. 

Resolved.